File: benchmark.go
package main

import (
	"fmt"
	"time"
)

func isPrime(n int) bool {
	for i := 2; i < n/2+1; i++ {
		if n%i == 0 {
			return false
		}
	}

	return true
}

func main() {
	start := time.Now()

	var numPrimes int

	for i := 2; i < 250001; i++ {
		if isPrime(i) {
			numPrimes++
		}
	}

	fmt.Printf("Computed %v primes in %v\n", numPrimes, time.Since(start))
}

File: benchmark.py
#!/usr/bin/env python3
import time

def isPrime(n: int) -> int:
    for i in range(2,n//2+1):
        if n%i == 0:
            return 0

    return 1

if __name__ == "__main__":
    start = time.time()

    numPrimes = 0

    for i in range(2,250001):
        numPrimes+=isPrime(i)

    finish = time.time()
    print(f"Computed {numPrimes} primes in {finish - start}s")